addons/website_slides/static/src/views/slide_channel_partner_list/slide_channel_partner_list_controller.js

```diff
@@ -0,0 +1,35 @@
+/** @odoo-module **/
+
+import { ListController } from '@web/views/list/list_controller';
+import { useService } from '@web/core/utils/hooks';
+
+
+export default class SlideChannelPartnerListController extends ListController {
+    setup() {
+        super.setup();
+        this.action = useService('action');
+        this.orm = useService('orm');
+        this.channelId = this.props.context.default_channel_id || false;
+    }
+
+    /**
+     * Method opening the wizard to enroll new slide channel partners.
+     * Reloads the model afterwards to see new attendees.
+     * 
+     * @private
+     */
+    async _openEnrollWizard() {
+        const action = await this.orm.call(
+            'slide.channel',
+            'action_channel_enroll',
+            ['', this.channelId]
```

Comment:
Why don't you do 
```
this.orm.call(
            'slide.channel',
            'action_channel_enroll',
            [this.channelId]
``` ?

Reply:
Because before, we allowed the user to change the course. I'm changing that :)